app/models/tslr_claim.rb

```diff
@@ -39,21 +39,21 @@ class TslrClaim < ApplicationRecord
 
   validates :mostly_teaching_eligible_subjects, on: [:"mostly-teaching-eligible-subjects", :submit], inclusion: {in: [true, false], message: "Select either Yes or No"}
 
-  validates :full_name,                         on: [:"full-name", :submit], presence: {message: "Enter your full name"}
+  validates :full_name,                         on: :submit, presence: {message: "Enter your full name"}
```

Comment:
These messages should probably be updated to ask the user to re-Verify, or something. Asking a user to enter their name, but not giving them any mechanism to do so is a road to frustration. Though what would cause these fields to be missing that the user could do anything about anyway?

Reply:
Good point, I'll raise this at stand-up tomorrow

Reply:
I totally forgot about this at stand-up. I'd be interested to know your thoughts @titlescreen @tekin 

I think we need to keep the validation on `submit` for these values as it's what makes a claim "submittable". I don't think a user should be able to get in a state where they'll see this error message because we get their details from Verify (are we confident that we always get all the fields?). 

I'm not sure about the error message, we could go with something more generic like "Full name is required" for the cases where we might manually create/update claim records.

Reply:
Technically a user could jump to /check-your-answers before going through verify, or at least they can at the moment, but yes, in theory a user going through the normal flow shouldn't ever see this message. It might be worth adding something to the backlog so that we prevent users moving beyond the verify step until they've actually got through verify.

Reply:
So perhaps it makes sense to rephrase the error message to ask the user to (re-)Verify, and make the "change" links take them to the Verify step again (though they'd need some way to decide _not_ to re-Verify other than their browser back button)?

Reply:
I had a chat with Rich about this and we feel the re-verification step needs some design thinking. Maybe it needs to happen as soon as the user is returned from Verify?

We know that we'll get back name, address and date of birth from Verify so it feels like an edge case and if it is an issue it should surface during private beta.

I agree that we should prevent users from skipping ahead to the check-your-answers page.